assets/js/modules/analytics-4/components/settings/AnalyticsSettingsNotice.js

```diff
@@ -47,10 +45,10 @@ export default function AnalyticsSettingsNotice( { hasModuleAccess } ) {
 
 	if ( ! hasModuleAccess ) {
 		return (
-			<SettingsNotice
-				type={ TYPE_INFO }
-				Icon={ WarningIcon }
-				notice={ createInterpolateElement(
+			<Notice
+				className="googlesitekit-settings-notice"
```

Comment:
Do we really need this class? If yes, could we create a new class with the bare minimum styles needed to style a `Notice` within settings? We should put any such special wrapper classes as variations within `googlesitekit-notice.scss` or in the `googlesitekit-settings-module.scss` file or the relevant stylesheet for this particular component. 

Currently, the `googlesitekit-settings-notice` class has loads of legacy styles which ideally, we want to remove fully when we get rid of the `SettingsNotice` react component!

Reply:
@jimmymadon I had a look, and this is used by a few modules to ensure that there are no margins (top, bottom or other) based on where they show, and it seems to be module specific. I am hesitant to remove this at this stage, as it is probably being used by no longer refactored items. From the VRTs, I am not seeing any negative visual impact of using this. Perhaps, once all notice types are refactored, we should have a follow up to assess the usage and removal of this?